tutorials/abap-environment-text-table/abap-environment-text-table.md

```diff
@@ -279,59 +310,66 @@ Many business configuration objects use text tables for providing translatable a
 
   2. Save and activate.
 
-  3. Let's now continue and directly add the annotations for table display. Open your text table CDS view **`ZCAL_I_HOLITXT_XXX`**.
+  3.  Open **`ZCAL_I_HOLITXT_XXX`** and add line times
```

Comment:
line times - Is this a typo and should be "line items"?

Reply:
Yes, it is. Should I change it?